truss/truss_handle/truss_handle.py

```diff
@@ -309,12 +309,9 @@ def docker_run(
                 envs["PATCH_PING_URL_TRUSS"] = patch_ping_url
             if disable_json_logging:
                 envs["DISABLE_JSON_LOGGING"] = "true"
-            if self.spec.config.docker_server and self.spec.config.training_checkpoints:
```

Comment:
Related to a previous discussion, but we shouldn't need these lines after L315 below